File: app/src/main/kotlin/za/foundation/praekelt/mama/api/db/AppDatabase.java
package za.foundation.praekelt.mama.api.db;

import com.raizlabs.android.dbflow.annotation.Database;

import za.foundation.praekelt.mama.util.Constants;

/**
 * Application database object
 */

@Database(name = AppDatabase.NAME, version = AppDatabase.VERSION, generatedClassSeparator = "_")
public class AppDatabase {
    public static final String NAME = Constants.REPO_NAME;
    public static final int VERSION = 1;
}

File: app/src/main/kotlin/za/foundation/praekelt/mama/api/db/AppDatabase.kt
package za.foundation.praekelt.mama.api.db

import com.raizlabs.android.dbflow.annotation.Database
import za.foundation.praekelt.mama.util.Constants as _C

/**
 * Application database object
 */

@Database(name = AppDatabase.NAME, version = AppDatabase.VERSION, generatedClassSeparator = "_")
public object AppDatabase{
    public const val NAME: String = _C.REPO_NAME
    public const val VERSION: Int = 1
}